Eternalgratis/Math-magician
# Math Magicians: tests

### Math Magicians: tests

* Set up React Testing Library.
* Follow the instructions in the official [documentation](https://testing-library.com/docs/react-testing-library/intro).
* You don't need to setup Jest (it's included in Create React App).
* Write unit tests for the files operate.js and calculate.js using Jest.
* Create unit tests for all React components:
* Use Jest snapshots to test the components.
* Use React Testing Library to simulate user interaction.
* Run your full tests suite using Jest. All tests should pass.
